### PR TITLE
move gg certs into secrets manager

### PR DESCRIPTION
**Pull request by thomas-roos**
putting gg certs into secrets manager, for this permissions must be granted. This is planned to add as a feature to https://github.com/aws4embeddedlinux/aws4embeddedlinux-ci

### CODE REVIEW

**Comment by thomas-roos** [comment]
As changes are applied to [aws4embeddedlinux-ci](https://github.com/aws4embeddedlinux/aws4embeddedlinux-ci) this can be merged.

File: cdk/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "meta-aws-cdk-pipeline-reference",
+  "name": "demo-iot-automotive-embeddedlinux-image-pipeline",
```
[the remark this answers is not in the document]
Comment (thomas-roos) [reply cdk/package.json:2]: yw!

File: meta-aws-iot-automotive-demo/recipes-core/images/aws-biga-image/aws-biga-image.bb [hunk 2]
```diff
@@ -156,3 +156,4 @@ IMAGE_INSTALL += "netcat"
 IMAGE_INSTALL += "curl"
 
 IMAGE_INSTALL += "ntp-systemd"
+
```
Comment (alinadima) [inline meta-aws-iot-automotive-demo/recipes-core/images/aws-biga-image/aws-biga-image.bb:159]: Is this new line needed?

**Review by alinadima — APPROVED** [review]

File: meta-aws-iot-automotive-demo/recipes-core/images/aws-biga-image/aws-biga-image.bb (hunk 2 above)
Comment (thomas-roos) [reply meta-aws-iot-automotive-demo/recipes-core/images/aws-biga-image/aws-biga-image.bb:159]: You're right there is no reason for this. In some cases it is just good practice, but I found nothing about this on bitbake style guide etc.
Will remove this file from PR

**Pull request merged**